web/js/util/util.js

```diff
@@ -1,16 +1,21 @@
-/**
- * @module wv.util
- */
-var wv = wv || {};
-
-/**
- * General utilities
- *
- * @class wv.util
- * @static
- */
-wv.util = (function (self) {
+/* global ntptEventTag */
+
+import $ from 'jquery';
+import { each, isNull } from 'lodash';
```

Comment:
do we want to use a syntax for lodash like `_each` so we know that we are using a lodash method or do we want to just include the whole package to avoid the hassle? @localjo @ZachTRice 

I was working with one file where I was importing 4+ lodash methods so I decided to use the whole package. I could change that though...

I think we should definitely import  the whole `openlayers ` package

Reply:
Hm, I have been importing separate lodash functions, but agree we should probably make them unique by appending `_` to the function name if we are doing this.

Reply:
We should be importing only the functions we need from `lodash` and any other dependencies that support it. That keeps our bundle size as small as possible.  Even if we're using 4+ lodash methods, importing the whole library blows up bundle sizes.

As far as naming goes, it's all within the scope of the file, so it's easy to see where the function names are defined at the top of the file. We could rename them like `_each`, but really, that's unnecessary.